ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/simulation/sensors/RDXSensorSimulator.java

```diff
@@ -173,19 +175,9 @@ public void render(RigidBodyTransform cameraTransformToWorld)
          opencv_core.flip(colorImage, colorImage, OpenCVTools.FLIP_Y);
       }
 
-      if (depthImage != null) // If depth is enabled
+      if (depthImage != null)
       {
-         // Render and "grab" the color image
-         simulationScene.preRenderDepth(camera);
-         GL41.glViewport(0, 0, imageWidth, imageHeight);
-         simulationScene.renderDepth(RDXSceneLevel.GROUND_TRUTH.SINGLETON_SET);
-         simulationScene.postRenderDepth();
```

Comment:
The color render generates the depth data we are using. We don't have to render again for depth.

Reply:
The point here is that it only renders for the ground truth scene level, so excluding virtual and modelled renderables. Virtual -> gizmos, etc.   Modelled -> point cloud, height map, etc.

Reply:
I think the color render also only renders for the ground truth scene level. Perhaps I have to double check that, but I'd be surprised if gizmos are visible in our simulated sensors.

Reply:
I took another look at the code and can confirm that the color render also only renders ground truth.